src/Config.php

```diff
@@ -123,4 +129,14 @@ private function parseConfObject($conf_object)
         $affinity_conf['affinity_by_method'] = $aff_by_method;
         return $affinity_conf;
     }
+
+    private function get_shmem() {
```

Comment:
This should follow the convention in the rest of the file:

```php
private function get_shmem()
{
```

Reply:
done.